UK_Test-Trace_phaseplots_26August.py

```diff
@@ -252,7 +253,8 @@
     except:
         pass
     outfile = "%s/test%s-trace%s.obj" % (scenario, args.test, args.trace)
-    sc.saveobj(outfile, sc.objdict((("args", args), ("results", msim.results))))
+    msim.args = args
+    sc.saveobj(outfile, sc.objdict((("args", args), ("msim", msim.results), ("sims", list(sim.results for sim in msim.sims)))))
```

Comment:
Since dicts in Python 3.6+ are ordered, you can save a tiny bit of space and a lot of parentheses with
```
sc.saveobj(outfile, sc.objdict({"args":args, "msim":msim.results, "sims":list(sim.results for sim in msim.sims)}))
```